<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsCOM</class>
 <widget class="QMainWindow" name="SettingsCOM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxCOM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>COM port</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxShpindle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Spindle</string>
    </property>
    <widget class="QSpinBox" name="spinBoxDelayAfterON">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Задержка после включения</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Задержка после выключения</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDelayAfterOFF">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxNumbOfTurns">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. число оборотов</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="applyButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canselButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cansel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveSettingsButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSound">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Sound</string>
    </property>
    <widget class="QCheckBox" name="checkBoxSoundAtEndProgramm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>По окончанию программы</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSoundAtEndCycle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>По завершению цикла</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSoundAtChangeTool">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Смена инструмента</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSoundAtChangeToolRepeat">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>повторять с периодом 5 сек.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxLaser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Laser</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxAxis">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Axis</string>
    </property>
    <widget class="QSpinBox" name="spinBoxFastMove">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxSlowMove">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ускоренное перемещение</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Перемещение подачи</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>canselButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsCOM</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
